MultiServer.py

```diff
@@ -649,7 +649,8 @@ def get_aliased_name(self, team: int, slot: int):
         else:
             return self.player_names[team, slot]
 
-    def notify_hints(self, team: int, hints: typing.List[NetUtils.Hint], only_new: bool = False):
+    def notify_hints(self, team: int, hints: typing.List[NetUtils.Hint], only_new: bool = False,
+                     recipients: typing.Sequence[int] = None):
```

Comment:
Should this is be typing.Optional[typing.Sequence[int]] = None ?

Reply:
No, = None is special cased as already implying Optional